Caprica/main_options.cpp

```diff
@@ -162,7 +162,10 @@ bool parseCommandLineArguments(int argc, char* argv[], caprica::CapricaJobManage
         "Do not report progress, only failures.")("strict",
                                                   po::value<bool>()->default_value(false)->implicit_value(true),
                                                   "Enable strict checking of control flow, poisoning, and more sane "
-                                                  "implicit conversions. It is strongly recommended to enable these.");
+                                                  "implicit conversions. It is strongly recommended to enable these.")(
+        "ignorecwd",
+        po::bool_switch()->default_value(false),
```

Comment:
Is `->implicit_value(true)` needed as well? Without it I believe you would have to explicitly pass `--ignorecwd=true`, rather than being able to just pass `--ignorecwd`
```suggestion
        po::bool_switch()->default_value(false)->implicit_value(true),
```